src/DeliveryService.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Drupal\delivery;
 
-use Drupal\Core\Conflict\ConflictResolver\ConflictResolverManagerInterface;
+use Drupal\conflict\ConflictResolver\ConflictResolverManager;
```

Comment:
This was provoking WSOD, probably updated on core to its own namespace. More instances below.